stream_alert_cli/terraform/monitoring.py

```diff
@@ -19,6 +19,19 @@
 def generate_monitoring(cluster_name, cluster_dict, config):
     """Add the CloudWatch Monitoring module to the Terraform cluster dict.
 
+    Example configuration:
+
+    "cloudwatch_monitoring": {
+      "enabled": true,
+      "kinesis_alarms_enabled": true,
+      "lambda_alarms_enabled": true,
+      "settings": {
+        "lambda_invocation_error_period": "600",
+        "kinesis_iterator_age_error_period": "600",
+        "kinesis_write_throughput_exceeded_threshold": "100"
```

Comment:
To be consistent with the rest of the configuration (kinesis `batch_size`, lambda `current_version`, etc), these should be numbers instead of strings. Terraform works fine with numbers, too.

Reply:
the documentation uses strings for all the alarms, I was just following their convention: https://www.terraform.io/docs/providers/aws/r/cloudwatch_metric_alarm.html